codespell_lib/data/dictionary.txt

```diff
@@ -15584,6 +15773,7 @@ unabale->unable
 unabel->unable
 unacceptible->unacceptable
 unaccesible->unaccessible
+unaccurate->inaccurate
```

Comment:
Obsolete, but not completely wrong. Maybe offer both variants.
https://en.wiktionary.org/wiki/unaccurate

Reply:
done

Reply:
As per https://github.com/codespell-project/codespell/issues/712 I really don't think we should have real words in the dictionary, add it to the list for that issue.

Reply:
Removed.